app/views/sessions/start.html.erb

```diff
@@ -38,7 +38,7 @@
 
       <% if @client_app  %>
           <% if @client_app.name.downcase.include? "tutor" %>
-            <div class="tutor-message">Log in or sign up for an account to access OpenStax Tutor</div>
+            <div class="tutor-message"><%= t '.tutor' %></div>
```

Comment:
To me calling the message just `tutor` is a bit too generic (though the `sessions` namespace does help I guess), I would suggest something like `access_tutor` at least.